common/src/main/java/com/viaversion/viabackwards/protocol/v1_21_6to1_21_5/provider/ChestDialogViewProvider.java

```diff
@@ -274,17 +281,22 @@ protected Item getItemWidget(final UserConnection connection, final ItemWidget i
     }
 
     protected Item getMultiTextWidget(final UserConnection connection, final MultiTextWidget multiTextWidget) {
-        final Tag name = handleTag(connection, multiTextWidget.labels()[0]);
-        final int length = multiTextWidget.labels().length;
-        if (length == 1) {
-            return createItem("minecraft:paper", name);
+        final List<Tag> lines = new ArrayList<>();
+        for (final Tag label : multiTextWidget.labels()) {
+            final Tag[] split = ChatUtil.split(fixStyle(label), "\n");
+            for (final Tag line : split) {
+                lines.add(handleTag(connection, line));
+            }
+        }
+        if (lines.isEmpty()) {
```

Comment:
this can be at the very top of the code and doesnt need the for loop I think, though is this really a condition that can happen? It wasn't handled before I think